client/src/components/filmBox.js

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useState, Fragment } from 'react';
 import FilmItem from './FilmItem';
+import Search from './Search';
+import CharacterCard from './CharacterCard';
+import posterUrls from '../static/imageAddresses';
 
 export default function FilmBox({ films }) {
+  const [extendedIndex, setExtendedIndex] = useState('');
```

Comment:
I understand. 
1- I set initial value to '' because I remember -1 means the last element of array, but I tried and with -1 there is no problem. I checked that and in slice and splice methods of array minus values mean sth. that's why I remember this way. 
2- better solutions / practises are important for me, so I changed it to id. thank you